src/main/java/core/basesyntax/behaviour/Suppliable.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.behaviour;
+
+public interface Suppliable {
```

Comment:
Consider renaming the interface 'Suppliable' to something more descriptive, such as 'FigureSupplier' or 'Randomizable'.